Fix: #1843. Improve example description. (#1982)
httpwg/http-extensions: draft-ietf-httpbis-digest-headers.md

"hello"
~~~
{: title="Partial response with both Content-Digest and Representation-Digest"}

> In the response message above, we note that
> the `Representation-Digest` field-value is calculated across the entire JSON object
> `{"hello": "world"}`, and the field will be

Reviewer: ```suggestion
In the response message above, note that the
`Representation-Digest` and `Content-Digests` are different.
The `Representation-Digest` field-value is calculated across the entire JSON object
`{"hello": "world"}`, and the field is
```
